src/interfaces/IRollsData.ts

```diff
@@ -1,3 +1,9 @@
+/**
+ * Represents the data of a roll.
+ *
+ * @see fee - fees paid for the roll
+ * @see amount - amount paid for the roll
```

Comment:
Can you confirm it's not the amount of rolls ?

Reply:
you are right